grafana/grafana.libsonnet

```diff
@@ -232,12 +232,23 @@ function(params) {
       mountPath: '/etc/grafana/provisioning/dashboards',
       readOnly: false,
     };
+    // A volume on /tmp is needed to let us use 'readOnlyRootFilesystem: true'
+    local pluginTmpVolume = {
+      name: 'tmp-plugins',
+      emptyDir: {},
```

Comment:
This would prevent node drain. I wonder if putting this into RAM would work better :thinking:

Reply:
Could you explain why it prevents node drain? I tried to find reason on k8s docs but got no luck 😬 

and is there a RAM type of volume? 🤔

Reply:
TIL :)

Changed it to Memory